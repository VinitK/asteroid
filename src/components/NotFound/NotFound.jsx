import React from 'react';
import { Link } from 'react-router-dom';

const NotFound = () => {
    return (
        <div className="NotFound w80pc ma">
            <h1>Asteroid Not Found</h1>
            <Link to="/" className="link button">Go Home</Link>
        </div>
    );
};

export default NotFound;