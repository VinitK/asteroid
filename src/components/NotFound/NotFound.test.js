import React from 'react';
import NotFound from './NotFound';
import { shallow } from 'enzyme';

describe('NotFound Tests', () => {
    let wrapper;
    beforeEach(() => {
        wrapper = shallow(<NotFound />);
    });
    it('NotFound Component Snapshot', () => {
        expect(wrapper).toMatchSnapshot();
    })
})