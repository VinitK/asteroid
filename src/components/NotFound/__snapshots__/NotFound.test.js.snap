// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound Tests NotFound Component Snapshot 1`] = `ShallowWrapper {}`;
